packages/flutter/lib/src/material/user_accounts_drawer_header.dart

```diff
@@ -43,41 +42,38 @@ class UserAccountsDrawerHeader extends StatefulWidget {
   /// section is pressed.
   final VoidCallback onDetailsPressed;
 
-  /// Decoration for the main drawer header container useful for applying
-  /// backgrounds.
-  final BoxDecoration decoration;
+  /// The background to show in the drawer header.
+  final Decoration decoration;
 
   /// A widget placed in the upper-left corner representing the current
   /// account picture. Normally a [CircleAvatar].
   final Widget currentAccountPicture;
 
-  /// A list of widgets that represent the user's accounts. Up to three of them
-  /// are arranged in a row in the header's upper-right corner. Normally a list
+  /// A list of widgets that represent the user's accounts. Up to three of will
+  /// be arranged in a row in the header's upper-right corner. Normally a list
   /// of [CircleAvatar] widgets.
   final List<Widget> otherAccountsPictures;
 
-  /// A widget placed on the top row of the account details representing
-  /// account name.
+  /// A widget placed on the top row of the account details representing the
+  /// account's name.
   final Widget accountName;
 
-  /// A widget placed on the bottom row of the account details representing
-  /// account email.
+  /// A widget placed on the bottom row of the account details representing the
+  /// account's e-mail address.
   final Widget accountEmail;
 
   @override
-  _UserAccountsDrawerHeaderState createState() =>
-      new _UserAccountsDrawerHeaderState();
+  _UserAccountsDrawerHeaderState createState() => new _UserAccountsDrawerHeaderState();
 }
 
 class _UserAccountsDrawerHeaderState extends State<UserAccountsDrawerHeader> {
-  /// Saves whether the account dropdown is open or not.
-  bool isOpen = false;
+  // Saves whether the account dropdown is open or not.
+  bool _isOpen = false;
```

Comment:
you know what, I just removed the entire comment. It's the state that says whether this object is open, so the meaning is pretty self-evident.